hack/make/.integration-test-helpers

```diff
@@ -15,36 +15,34 @@ source "$MAKEDIR/.go-autogen"
 : ${TESTFLAGS:=}
 : ${TESTDEBUG:=}
 
-integration_api_dirs=${TEST_INTEGRATION_DIR:-"$(
-	find ./integration -type d |
-	grep -vE '(^./integration($|/internal)|/testdata)')"}
+integration_api_dirs=${TEST_INTEGRATION_DIR:-"$(go list  -test -f '{{- if ne .ForTest "" -}}{{- .Dir -}}{{- end -}}'  ./integration/...)"}
 
 run_test_integration() {
 	set_platform_timeout
-	if [[ "$TESTFLAGS" != *-check.f* ]]; then
+	if [ -z "${TEST_SKIP_INTEGRATION}" ] && [[ "$TESTFLAGS" != *-check.f* ]]; then
```

Comment:
Could we somehow make `"$TESTFLAGS" != *-check.f*` set `$TEST_SKIP_INTEGRATION_CLI` (same for `TEST_SKIP_INTEGRATION`), so that further down the line, we'd only have to check for those variables?

Reply:
Done